acceptance/Rakefile

```diff
@@ -1,18 +1,26 @@
 require 'rake/clean'
 
+# Default test target if none specified
+DEFAULT_MASTER_TEST_TARGET = 'redhat7-64mdca'
+DEFAULT_TEST_TARGETS = "#{DEFAULT_MASTER_TEST_TARGET}-windows2012r2-64client."
+
 desc 'Run acceptance tests'
 
 desc "Generate Beaker Host config"
 task :host_config do
   if ENV["BEAKER_CONFIG"]
     next
   end
-  if not ENV['TEST_TARGET']
-    fail "FATAL: must set TEST_TARGET environment variable."
+
+  if client_target = ENV['TEST_TARGET']
+    master_target = ENV['MASTER_TEST_TARGET'] || DEFAULT_MASTER_TEST_TARGET
+    targets = "#{master_target}-#{client_target}client."
+  else
+    targets = DEFAULT_TEST_TARGETS
   end
 
   generate = "bundle exec beaker-hostgenerator"
-  generate += " --disable-default-role #{ENV['TEST_TARGET']}"
+  generate += " --disable-default-role #{targets}"
```

Comment:
This looks fine. However, I do not understand the purpose of the `JENKINS_HOME` conditional wrapper around the `BEAKER_PE_VER` conditional on line 31. Is the `JENKINS_HOME` check necessary?